src/commands/Preferences.js

```diff
@@ -9,27 +9,32 @@ export default function (context) {
     const preferences = getUserPreferences()
     var accessory = NSView.alloc().initWithFrame(NSMakeRect(0, 0, 300, 275))
```

Comment:
you need to increase the height here otherwise the first input `Folder where the pretty diffs will be exported` will disappear (see your screenshot)